config/source/1_engine/rex_engine/code_generation.json

```diff
@@ -27,11 +27,11 @@
     ],
     "Filepath": "1_engine/rex_engine/include/rex_engine/cmdline/cmd_args.h",
     "Content": [
-      "\"Root\", \"The directory to initialize the vfs with. this is similar as setting the working root directory\"",
-      "\"Help\", \"This will list all command line arguments for the executable\"",
-      "\"LogLevel\", \"Specify log level per logger (eg. -LogLevel=Engine=trace,Renderer=info)\"",
-      "\"BreakOnBoot\", \"Break on boot so you can attach a debugger\"",
-      "\"AttachOnBoot\", \"Attach the debugger on boot\""
+      "L\"Root\", L\"The directory to initialize the vfs with. this is similar as setting the working root directory\"",
+      "L\"Help\", L\"This will list all command line arguments for the executable\"",
+      "L\"LogLevel\", L\"Specify log level per logger (eg. -LogLevel=Engine=trace,Renderer=info)\"",
+      "L\"BreakOnBoot\", L\"Break on boot so you can attach a debugger\"",
+      "L\"AttachOnBoot\", L\"Attach the debugger on boot\""
```

Comment:
wdym? json doesn't know what the "L" means, this is so that the code generation, generates code (as in, uses wide chars), which makes it compile. otherwise I'd have to let the generation step take care of it and I don't think it belongs there